pkgs/by-name/ky/kyverno-chainsaw/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule (finalAttrs: {
+  pname = "kyverno-chainsaw";
+  version = "0.1.6";
+  cli = "chainsaw";
+
+  src = fetchFromGitHub {
+    owner = "kyverno";
+    repo = "chainsaw";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-14swkcv70qn1ka0pb3za9a4r15shm1aw93by2k3b1kx3qdar1bcs";
+  };
+
+  vendorHash = "sha256-466ad9e33cfd22ba799db99b21ab23d4";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildVersion=v${version}"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildHash=${version}"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildTime=1970-01-01_00:00:00"
+  ];
+
+  meta = with lib; {
+    description = "Chainsaw is a tool developed to run end to end tests in Kubernetes clusters.";
+    homepage = "https://github.com/kyverno/chainsaw";
+    changelog = "https://github.com/kyverno/chainsaw/releases/tag/v${finalAttrs.version}";
+    license = licenses.asl20;
```

Adding a long description could be helpful too and _if_ there is an appropriate place for listing adopters it could be this.

I'd avoid directly listing individual adopters in this package and would link to the project's [`ADOPTERS.md`](https://github.com/kyverno/chainsaw/blob/main/ADOPTERS.md) file. That said I'm not (yet?) convinced there is use for such kind of "advertising" within a nix package.

What is your motivation to include it, @Sanskarzz? If you could find/provide examples of other nix packages somehow including adopters that might be helpful.
```suggestion
    license = licenses.asl20;
    longDescription = ''
      Kubernetes policy engine and CNCF project that has the ability to
      validate, mutate, generate, and clean-up resources plus verify OCI container
      images for supply chain security guarantees.
    '';
```

---

Please wait i will just clarify this with kyverno/chainsaw maintainer .

---

As maintainer suggest We don't care about the list of adopters. Keep it simple ;)